index.md

```diff
@@ -95,6 +95,12 @@ by Yash and Karthik.
 
 Another [flashcard set](https://quizlet.com/mian8680/folders/bu288-f2021/sets){:.a#BU288--Flashcards--Imaad} for the two midterms and final exam.
 
+#### BU 288 (F23) <small>Organizational Behaviour</small>
+
+[Notes for Midterm 1 content](./BU288/f23-mt1-notes.pdf), [Notes for Midterm 2 content](./BU288/f23-mt2-notes.pdf) by Andy. Unfortunately a set for the finals do not exist from him.
```

Comment:
can you add the tracking classes? it's just fun to have data :)

Reply:
so thats what those do - should be added now

Reply:
needs to be double hyphens lol it's `course-code--category-name--author-name--title` where single hyphens get replaced by spaces. very stupid system that i cooked up at like 2am one night LOL

Reply:
i think i fixed it now?